lisa/runners/lisa_runner.py

```diff
@@ -75,7 +78,10 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self.environments: List[Environment] = [
             x for x in runbook_environments.values()
         ]
-        self._log.debug(f"candidate environment count: {len(self.environments)}")
+        self._log.debug(
+            f"Candidate environment count: {len(self.environments)}. "
+            f"Guest enabled, {self._guest_enabled}."
```

Comment:
f"Guest enabled: {self._guest_enabled}."

Reply:
how about this one @squirrelsc

Reply:
Missed, and fixed.